llvm/tools/llvm-exegesis/lib/BenchmarkRunner.cpp

```diff
@@ -175,6 +179,7 @@ class InProcessFunctionExecutorImpl : public BenchmarkRunner::FunctionExecutor {
   const LLVMState &State;
   const ExecutableFunction Function;
   BenchmarkRunner::ScratchSpace *const Scratch;
+  const std::optional<int> BenchmarkProcessCPU;
```

Comment:
Why do we have this member ? It's never read and the feature is not supported in this mode.